### PR TITLE
Remove deprecation warning

### PR DESCRIPTION
**Pull request by franmomu**
## Subject

When the bundle is loaded, [there is always a deprecation with the `sonata.twig.deprecated_template_extension` service](https://github.com/sonata-project/twig-extensions/runs/1070774891#step:11:132). 

Taking into account that by default services are private, and in this case is a Twig extension, I don't think the deprecation is needed, just a comment to remove the service in the next major version. 

I haven't checked, but I guess it throws the deprecation warning because it is tagged as `twig.extension`, so twig loads it anyway.

I am targeting this branch, because these changes are BC.

## Changelog

```markdown
### Fixed
- Fixed throwing a deprecation warning every time the bundle is loaded.
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
         "symfony/framework-bundle": "^4.4 || ^5.0",
```
Comment (jordisala1991) [inline composer.json:40]: can you change this one to 5.1 too?

Comment (franmomu) [reply composer.json]: Sure thing!

File: src/Bridge/Symfony/Resources/config/twig.xml [hunk 2]
```diff
@@ -1,25 +0,0 @@
...
-        </service>
-        <service id="sonata.twig.status_runtime" class="Sonata\Twig\Extension\StatusRuntime">
-            <tag name="twig.runtime"/>
-        </service>
-        <service id="sonata.twig.status_extension" class="Sonata\Twig\Extension\StatusExtension">
-            <tag name="twig.extension"/>
-        </service>
-        <!-- NEXT_MAJOR: Remove this definition. -->
-        <service id="sonata.twig.deprecated_template_extension" class="Sonata\Twig\Extension\DeprecatedTemplateExtension">
-            <deprecated>The "%service_id%" service is deprecated since sonata-project/twig-extensions 1.4.</deprecated>
```
Comment (jordisala1991) [inline src/Bridge/Symfony/Resources/config/twig.xml:16]: you removed this line, is that normal?

Comment (franmomu) [reply src/Bridge/Symfony/Resources/config/twig.xml:16]: Yes, see https://github.com/sonata-project/twig-extensions/pull/124#issue-482833284, maybe I should've split the PR into two. I've removed this service in the second commit https://github.com/sonata-project/twig-extensions/pull/124/commits/5af6ad552d4e3ae4cd7a03fcaaf9b8810d2af199.

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @franmomu !

**Pull request merged**